pkgs/by-name/sc/scotty/package.nix

```
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenv,
+  buildGoModule,
+  fetchFromSourcehut,
+  nix-update-script,
+  testers,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "scotty";
+  version = "0.4.1";
+
+  src = fetchFromSourcehut {
+    owner = "~phw";
+    repo = "scotty";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-R9MgX5Am//9ULNKWqfH/JthMhqg4FFitOds7wU5iDSU=";
+  };
+
+  # Otherwise checks fail with `panic: open /etc/protocols: operation not permitted` when sandboxing is enabled
+  # https://github.com/NixOS/nixpkgs/pull/381645#issuecomment-2656211797
+  preBuild = lib.optionalString stdenv.hostPlatform.isDarwin ''
+    substituteInPlace vendor/modernc.org/libc/honnef.co/go/netdb/netdb.go \
+      --replace-fail '!os.IsNotExist(err)' '!os.IsNotExist(err) && !os.IsPermission(err)'
+  '';
+
+  vendorHash = "sha256-wlyoN6y/Bh8f21d5bIzwRwWkGlXbFFx+uGK6c5PyKTs=";
+
+  env = {
+    # *Some* locale is required to be set
+    # https://git.sr.ht/~phw/scotty/tree/04eddfda33cc6f0b87dc0fcea43d5c4f50923ddc/item/internal/i18n/i18n.go#L30
+    LC_ALL = "C.UTF-8";
+  };
+
+  passthru = {
+    tests.version = testers.testVersion {
+      # See above
+      command = "LC_ALL='C.UTF-8' scotty --version";
+      package = finalAttrs.finalPackage;
+    };
+
+    updateScript = nix-update-script { };
+  };
+
+  meta = {
+    description = "Transfers your listens between various music listen tracking and streaming services.";
```

```suggestion
    description = "Transfers your listens between various music listen tracking and streaming services";
```
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description
> Not end with a period (or any punctuation for that matter).